engine/schema/src/com/cloud/host/dao/HostDaoImpl.java

```diff
@@ -144,6 +146,8 @@
     protected HostTransferMapDao _hostTransferDao;
     @Inject
     protected ClusterDao _clusterDao;
+    @Inject
+    private ConfigurationDao _configDao;
```

Comment:
@mrunalinikankariya please re-submit a PR that uses a ConfigKey based approach than use direct read/manipulation based on `configDao`.

Reply:
@rhtyd,

Created new PR2292 for config key based approach.